docs/examples/evaluation/TonicValidateEvaluators.ipynb

```diff
@@ -0,0 +1,399 @@
+{
```

Comment:
<p>This is great. I think this can be used to run on our <code>LabelledRagDatasets</code> which essentially carry a set of <code>LabelledRagDataExample</code>'s that carry <code>question</code> <code>reference_answer</code> and <code>reference_contexts</code> . You just need to call method <code>make_predictions_with(QueryEngine)</code> in order to get a <code>prediction</code> (i.e., <code>llm_answer</code>).</p><p> </p><p>Do you think you could add that example here:</p><p> </p><pre class="ql-syntax" spellcheck="false">!llamaindex-cli download-llamadataset EvaluatingLlmSurveyPaperDataset --download-dir ./data

from llama_index import SimpleDirectoryReader
from llama_index.llama_dataset import LabelledRagDataset
from llama_index import VectorStoreIndex

rag_dataset = LabelledRagDataset.from_json("./data/rag_dataset.json")
documents = SimpleDirectoryReader(
    input_dir="./data/source_files"
).load_data(num_workers=4)  # parallel loading

index = VectorStoreIndex.from_documents(documents=documents)
query_engine = index.as_query_engine()

# to get llm_answer
predictions_dataset = rag_dataset.make_predictions_with(query_engine)

# create your lists
questions, retrieved_context_list, reference_answers, llm_answer = zip(*[
   (e.query, e.reference_contexts, e.reference_answer, p.response) for e,p in zip(rag_dataset.examples, predictions_dataset.predictions)
])
</pre><p>          </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_index/pull/10000/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
Our `LabelledRagDatasets` btw can be downloaded from [llamahub](https://llamahub.ai/?tab=llama_datasets)

Reply:
This dataset is awesome, thanks for pointing it out to me! I added a section to the example notebook that uses this dataset as an example with the code snippet you sent.

Reply:
Awesome - thank you so much!